instagram/src/components/CommentSection/commentSection.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import './commentsection.css';
+import Comment from './Comment';
+import PropTypes from 'prop-types';
+
+class CommentSection extends React.Component {
+	state = {
+		comments: [],
+		newCommentText: '',
+		likes: 0
+	};
+
+	componentDidMount() {
+		this.setState({ comments: this.props.comments, likes: this.props.likes });
+	}
+
+	onChange = e => {
+		this.setState({ newCommentText: e.target.value });
+	};
+
+	addComment = () => {
+		const username = 'Rob';
+		const text = this.state.newCommentText;
+		this.setState({
+			comments: [...this.state.comments, { username, text }],
+			newCommentText: ''
+		});
+	};
+
+	likePost = () => {
+		this.setState({ likes: this.state.likes + 1 });
+	};
+	render() {
+		return (
+			<div className="comments-section">
+				<img
+					onClick={this.likePost}
+					className="loveheart"
+					src={require('../assets/loveheart.png')}
+					alt={'loveheart'}
+				/>
+				<img
```

Comment:
I'd consider reworking the way a comment gets added here. Maybe a `form` with an `input` element? From the user's perspective, it's not clear that they need to click on the comment bubble to submit their comment after they enter it into the field.